script.js

```diff
@@ -1,4 +1,103 @@
+var myOutputValue = "";
+var userWin = 0;
+var computerWin = 0;
+var totalPlay = 0;
+var currentGameMode = "Waiting for username";
+var userName = "";
+
+// Computer generates random Number
+var randomNumber = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  return randomInteger;
+};
+
+//Computer assigns number to "scissors", "paper", or "stone"
+var computerGuesses = function () {
+  var computerGuess = "";
+  var guess = randomNumber();
+  if (guess == 0) {
+    computerGuess = "scissors";
+  }
+  if (guess == 1) {
+    computerGuess = "paper";
+  }
+  if (guess == 2) {
+    computerGuess = "stone";
+  }
+  return computerGuess;
+};
+
+// User input "scissors", "paper", or "stone";
+// if user input beats computer's, user win
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var userInput = input.toLowerCase();
```

Comment:
Love that you used .toLowerCase() to allow for variations of the input to be accepted!